scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -1234,7 +1234,7 @@ BEGIN
     AND c.oid = reloid
     AND am.amname != 'gist'
   LOOP
-    sql := Format('CREATE INDEX %s_%s_gix ON %s USING GIST (%s)', relname, rec.attname, reloid::text, rec.attname);
+    sql := Format('CREATE INDEX ON %s USING GIST (%s)', reloid::text, rec.attname);
```

Comment:
It’s happening w/i the context of a loop that is finding only geometry columns that don’t already have gist indexes. So as long as the table rename retained the reference to the previous indexes, no extra indexes will get created. 